lib/run_loop/xcrun.rb

```diff
@@ -59,7 +59,7 @@ def exec(args, options={})
         command_output = CommandRunner.run(['xcrun'] + args, timeout: timeout)
 
         if command_output[:out]
-          out = command_output[:out].force_encoding('utf-8').chomp
+          out = command_output[:out].encode('UTF-8', 'UTF-8', :invalid => :replace).chomp
```

Comment:
Gotta have some rspec tests for this somehow.

I would mock the CommandRunner output.

```
let (:process_status) do
   Class.new do
      def to_s; '#<Mock ProcessStatus>'; end
      def inspect; to_s; end
      def exitstatus; 0; end
    end.new
end

it "handles UTF-8 encoding" do
args = ["xcrun", "sample"]
options = { :timeout => 2.0 }
hash = {
   :out => "some string with some non-UTF8 character that will be replaced",
   :pid => 11,
   :status => process_status.new
}
expect(CommandRunner).to receive(:run).with(args, options).and_return(hash)

actual = Xcrun.new.exec("sample")

expect(actual[:out]).to be == "string with unknown UTF-8 character replaced with `''`"
end
```